easybuild/easyconfigs/a/AGAT/AGAT-0.9.2-GCC-11.2.0.eb

```diff
@@ -0,0 +1,67 @@
+# easybuild easyconfig
+#
+# John Dey <jfdey@fredhutch.org> Fred Hutchinson Cancer Center
+#
+easyblock = 'Bundle'
+
+name = 'AGAT'
+version = '0.9.2'
+
+homepage = 'https://agat.readthedocs.io/en/latest/'
+description = """AGAT: Another GTF/GFF Analysis Toolkit. Suite of tools to handle gene annotations
+ in any GTF/GFF format."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+builddependencies = [('binutils', '2.37')]
+
+dependencies = [
+    ('Perl', '5.34.0'),
+    ('BioPerl', '1.7.8'),
+]
+
+exts_defaultclass = 'PerlModule'
+exts_filter = ("perldoc -lm %(ext_name)s ", "")
```

Comment:
```suggestion
exts_filter = ("perl -e 'require %(ext_name)s'", '')
```

Reply:
@fizwit thanks for adding the sanity_check_commands.
I made the above change for you, as this is a more robust way to check if a perl module is correctly installed.